### PR TITLE
Spiked Wristbands

### PR DESCRIPTION
**Pull request by EmilyR42**
- Added new Spiked Wristbands item with options to wear one on either arm
- Tweaked a few existing items to hide bracelets

### CODE REVIEW

File: BondageClub/Assets/Female3DCG/Female3DCG.js [hunk 1]
```diff
@@ -1141,12 +1141,17 @@ var AssetFemale3DCG = [
         Priority: 46,
         Default: false,
 		Clothing: true,
-        AllowPose: ["Yoked", "OverTheHead"],
+        AllowPose: ["TapedHands", "Yoked", "OverTheHead", "BackCuffs" ],
```
Comment (ace-1331) [inline BondageClub/Assets/Female3DCG/Female3DCG.js:1144]:
```suggestion
        AllowPose: ["Yoked", "OverTheHead", "BackCuffs" ],
```

**Review by ace-1331 — CHANGES_REQUESTED** [review]

**Comment by ace-1331** [comment]
note that this PR will conflict with #2786 and whichever gets merged last will need to regroup their item under the same Bracelet group in the extended file

**Review by ace-1331 — DISMISSED** [review]

**Comment by Ben987** [comment]
This one creates a few conflicts on github.  Can you resolve them and test again?  Thanks a lot :)

File: BondageClub/Assets/Female3DCG/Female3DCGExtended.js [hunk 2]
```diff
@@ -3619,7 +3648,7 @@ var AssetFemale3DCGExtended = {
 				},
 			},
 		}, // WebBlindfold
-	}, // ItemHead
+	}, // ItemHead
```
Comment (ace-1331) [inline BondageClub/Assets/Female3DCG/Female3DCGExtended.js:3651]:
```suggestion
	}, // ItemHead
```

**Review by ace-1331 — APPROVED** [review]

**Review by gatetrek — APPROVED** [review]

**Pull request merged**